src/components/Grid/methods.js

```diff
@@ -983,6 +989,7 @@ export default {
     this.axis.y.style('visibility', this.gridVisible ? 'visible' : 'hidden');
   },
   updateGrid() {
+    window.theGrid = this;
```

Comment:
What is this reference for? Do we want to keep it?

Reply:
Nope, that was for debugging. Thanks for catching it.